tests/serverpod_test_server/test/defaults/string/string_default_persist_test.dart

```diff
@@ -21,6 +21,94 @@ void main() async {
           expect(object.stringDefaultPersist, equals('A specific value'));
         },
       );
+
```

Comment:
Question: What are we validating with these additional tests that are not validated with the first ones?